tests/testthat/test-stanmodule.R

```diff
@@ -148,25 +154,9 @@ test_that("loading multiple lines from a file works as expected", {
         actual@generated_quantities,
         c("mystring; mystring2\nmystring3;\nmystring4;", "more strings")
     )
+})
 
 
-    actual_char <- as.character(actual)
-    expected <- paste0(
-        c(
-            "",
-            "generated quantities {",
-            "mystring; mystring2",
-            "mystring3;",
-            "mystring4;",
-            "more strings",
-            "}",
-            ""
-        ),
-        collapse = "\n"
-    )
-    expect_equal(actual_char, expected)
-
-})
```

Comment:
@G-Kazantzidis , How come this was removed sorry ?

Reply:
Apologies for that. I might have deleted it by mistake when I changed the implementation of `as.character`. I put it back now.